static/app/views/replays/detail/errorList/index.tsx

```diff
@@ -0,0 +1,195 @@
+        }) =>
+          rowIndex === 0 ? (
+            <ErrorHeaderCell
+              ref={e => e && registerChild?.(e)}
+              handleSort={handleSort}
+              index={columnIndex}
+              sortConfig={sortConfig}
+              style={{...style, height: HEADER_HEIGHT}}
+            />
+          ) : (
+            <ErrorTableCell
+              columnIndex={columnIndex}
+              currentHoverTime={currentHoverTime}
+              currentTime={currentTime}
+              handleMouseEnter={handleMouseEnter}
+              handleMouseLeave={handleMouseLeave}
+              onClickTimestamp={handleClick}
+              ref={e => e && registerChild?.(e)}
+              rowIndex={rowIndex}
+              sortConfig={sortConfig}
+              crumb={error}
+              startTimestampMs={startTimestampMs}
+              style={{...style, height: BODY_HEIGHT}}
+            />
+          )
+        }
+      </CellMeasurer>
+    );
+  };
+
+  return (
+    <FluidHeight>
+      <ErrorFilters errorCrumbs={errorCrumbs} {...filterProps} />
+      <ErrorTable>
+        {errorCrumbs ? (
+          <OverflowHidden>
+            <AutoSizer onResize={onWrapperResize}>
+              {({height, width}) => (
+                <MultiGrid
+                  ref={gridRef}
+                  cellRenderer={cellRenderer}
+                  columnCount={COLUMN_COUNT}
+                  columnWidth={getColumnWidth(width)}
+                  deferredMeasurementCache={cache}
+                  estimatedColumnSize={100}
+                  estimatedRowSize={BODY_HEIGHT}
+                  fixedRowCount={1}
+                  height={height}
+                  noContentRenderer={() => (
+                    <NoRowRenderer
+                      unfilteredItems={errorCrumbs}
+                      clearSearchTerm={clearSearchTerm}
+                    >
+                      {t('No errors! Go make some.')}
+                    </NoRowRenderer>
+                  )}
+                  onScrollbarPresenceChange={onScrollbarPresenceChange}
+                  overscanColumnCount={COLUMN_COUNT}
+                  overscanRowCount={5}
+                  rowCount={items.length + 1}
+                  rowHeight={({index}) => (index === 0 ? HEADER_HEIGHT : BODY_HEIGHT)}
+                  width={width}
+                />
+              )}
+            </AutoSizer>
+          </OverflowHidden>
+        ) : (
+          <Placeholder height="100%" />
+        )}
+      </ErrorTable>
+    </FluidHeight>
+  );
+}
+
+const OverflowHidden = styled('div')`
+  position: relative;
+  height: 100%;
+  overflow: hidden;
+`;
+
+const ErrorTable = styled(FluidHeight)`
+  border: 1px solid ${p => p.theme.border};
+  border-radius: ${p => p.theme.borderRadius};
+
+  .beforeHoverTime + .afterHoverTime:before {
+    border-top: 1px solid ${p => p.theme.purple200};
+    content: '';
+    left: 0;
+    position: absolute;
+    top: 0;
+    width: 999999999%;
```

Comment:
👀 what does this do?

Reply:
This is a cool css hack for sure. It's already part of the network table.

This is drawing the purple line under each row to represent `currentTime` or `currentHoverTime`. Before we were passing in args to the `styled()` call on each render.

Now we set either `.beforeCurrentTime` or `.afterCurrentTime`  class on each cell, and then can use css sibling selectors to draw the line, so no `styled()` stuff needs to re-compute. All cells need to render with the css, because we might not have the first or last column visible on the screen, because the line draws really wide across the width of the whole table things look great